### DISCUSSION TITLE
luci-app-filetransfer 怎么没有了？

### DISCUSSION THREAD

**Discussion by Travel2Here** [coolsnowwolf/lede#12980] [Q&A / 快速问答]
原本库里默认有这个工具，为什么近来没有啦，是有什么兼容问题吗？
我感觉很好用啊，稳当一个版本的系统后就主要用这方式更新app了。

**Comment by xingshu1990** [comment]
https://github.com/coolsnowwolf/lede/issues/12609

**Reply by Travel2Here** [reply]
原来如此 多谢

**Reply by xingshu1990** [reply]
> 原来如此 多谢

不客气
